cfme/web_ui/__init__.py

```diff
@@ -3056,6 +3056,10 @@ class DynamicTable(Pretty):
     """
     pretty_attrs = "root_loc", "default_row_item"
     ROWS = ".//tbody/tr[not(contains(@id, 'new_tr'))]"
+    DELETE_ALL = {
```

Comment:
All means it is used for deleting all the fields, since it references possibly all the Minus buttons that are used to delete stuff.